swagger.yaml

```diff
@@ -2510,6 +2566,19 @@ definitions:
         type: string
         description: Json formatted string containing the user overrides for the host's pointer ignition
         example: '{"ignition": {"version": "3.1.0"}, "storage": {"files": [{"path": "/tmp/example", "contents": {"source": "data:text/plain;base64,aGVscGltdHJhcHBlZGluYXN3YWdnZXJzcGVj"}}]}}'
+      installer_args:
+        type: string
```

Comment:
Check that this is not a limited length value.

Reply:
Still need to fix it

Reply:
So it turns out that the default type for "string" is "text" :tired_face: 
ref: https://github.com/jinzhu/gorm/blob/466b344ff5924029ad8426073ba5064093e18a22/dialect_postgres.go#L51-L60

Also, just because we talked about it briefly and it's probably interesting for anyone reading this, [here's the documentation](https://www.postgresql.org/docs/13/datatype-character.html) on postgres character data types. It specifically calls out that there is no performance different in using variable length types vs text types.

That all said, I think I'll leave this to avoid confusion in the future. Maybe we can remove all of the gorm tags on string types in a cleanup PR.

Reply:
So it probably was different in mariadb that we used before postgres.